### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by Jatinkumar30**
Thanks for helping me getting grip on Open source and github.

### CODE REVIEW

**Review by prathamesh-borse — CHANGES_REQUESTED** [review]
Hey, @Jatinkumar30 You Should Not add the Space in between the GitHub Link. 
Because In Readme that Will Be Shown Your GitHub Profile when User Click on it.

File: README.md [hunk 1]
```diff
@@ -165,7 +165,7 @@ Hurray! You successfully made a contribution!
 
 - ### **J**
 
-  - [Jatin Kumar](https://github.com/Jatinkumar30)
+  - [Jatin Kumar]( https://github.com/Jatinkumar30 )
```
Comment (prathamesh-borse) [inline README.md:168]: @Jatinkumar30 Don't Add a Space between the GitHub Profile Link.

**Review by prathamesh-borse — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Jatinkumar30) [reply README.md]: Sure , I've removed the spacing now.

File: README.md [hunk 2]
```diff
@@ -165,7 +165,7 @@ Hurray! You successfully made a contribution!
 
 - ### **J**
 
-  - []()
+  - [Jatin Kumar](https://github.com/Jatinkumar30)
```
Comment (prathamesh-borse) [inline README.md:168]: Your PR looks good.
@Jatinkumar30 Thanks for making your contribution to the open-source world and stepping your feet in the open-source community.
Happy Hacking.! 👍 💯

**Review by prathamesh-borse — APPROVED** [review]

**Pull request merged**